### PR TITLE
not load legacy camera binary for ivi build

### PR DESCRIPTION
**Pull request by shivasku82**

### CODE REVIEW

**Comment by neofang7** [comment]
Looks good to me

**Review by balajim001 — APPROVED** [review]
LGTM

**Review by neofang7 — APPROVED** [review]

**Review by JeevakaPrabu — CHANGES_REQUESTED** [review]
If the service is intended for mediation or other purpose, isn't it better to move it to new mixins instead of having the product check here? mixins is intended for this

**Comment by balajim001** [comment]
vendor.camera.external=VHAL is this property rw ?

**Review by balajim001 — APPROVED** [review]
LGTM

File: groups/camera-ext/ivi/product.mk [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+                              ro.vendor.camera.in_frame_format.h264=false \
+                              ro.vendor.camera.in_frame_format.i420=true \
+                              ro.vendor.camera.decode.vaapi=false \
+                              ro.vendor.remote.sf.back_camera_hal= \
+                              ro.vendor.remote.sf.front_camera_hal= \
+                              ro.vendor.camera.transference="VSOCK" \
+                              vendor.camera.external="VHAL"
+#removing not required apps
+# Only include test apps in eng or userdebug builds.
+#PRODUCT_PACKAGES_DEBUG += TestingCamera
```
Comment (JeevakaPrabu) [inline groups/camera-ext/ivi/product.mk:22]: This can be under TARGET_BUILD_VARIANT check and included for eng, userdebug or removed completely

**Pull request merged**